test/metabase/api/dashboard_test.clj

```diff
@@ -3119,6 +3123,47 @@
                                                               :target [:dimension [:field (mt/id :venues :id) nil]]
                                                               :value  4}]})))))))))
 
+;; see also [[metabase.query-processor.dashboard-test]]
+(deftest dashboard-native-card-query-parameters-test
```

Comment:
Note this is like `dashboard-card-query-parameters-test` above, but for native query cards and with a non-required default